src/components/QuestionAnswer.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from 'react';
+import { subscribe } from './Event';
+import { getQuestion } from './Helper';
+
+function QuestionAnswer(props) {
+  const { questionNum, nextQuestion } = props;
+  const [questionObj, setQuestion] = useState(getQuestion(questionNum));
+  const [inputVal, setInputVal] = useState('');
+  useEffect(() => {
+    subscribe('NextQuestion', () => {
```

Comment:
Lets see if we can have an alternate to this custom event. This is also fine.

Reply:
Done